2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Configure stats pid to 1554373
2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Loading settings from /home/vb2386/.config/wandb/settings
2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Loading settings from /scratch/vb2386/Documents/nerf-pytorch/wandb/settings
2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'Sweep_Nerf', 'entity': 'vb2386', 'root_dir': '/scratch/vb2386/Documents/nerf-pytorch', 'sweep_id': 'yy5df601', 'run_id': 'cge0nt76', 'sweep_param_path': '/scratch/vb2386/Documents/nerf-pytorch/wandb/sweep-yy5df601/config-cge0nt76.yaml'}
2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-09 22:05:14,930 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'nerf-pytorch/profile_nerf.py', 'program_abspath': '/scratch/vb2386/Documents/nerf-pytorch/profile_nerf.py', 'program': 'profile_nerf.py'}
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_init.py:_log_setup():521] Logging user logs to /scratch/vb2386/Documents/nerf-pytorch/wandb/run-20240509_220514-cge0nt76/logs/debug.log
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_init.py:_log_setup():522] Logging internal logs to /scratch/vb2386/Documents/nerf-pytorch/wandb/run-20240509_220514-cge0nt76/logs/debug-internal.log
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_init.py:init():561] calling init triggers
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_init.py:init():568] wandb.init called with sweep_config: {'batch_size': 2048, 'chunk': 32768, 'lrate': 0.01}
config: {}
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_init.py:init():611] starting backend
2024-05-09 22:05:14,931 INFO    Thread-4  :1554373 [wandb_init.py:init():615] setting up manager
2024-05-09 22:05:14,941 INFO    Thread-4  :1554373 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-09 22:05:14,943 INFO    Thread-4  :1554373 [wandb_init.py:init():623] backend started and connected
2024-05-09 22:05:14,951 INFO    Thread-4  :1554373 [wandb_run.py:_config_callback():1347] config_cb None None {'batch_size': 2048, 'chunk': 32768, 'lrate': 0.01}
2024-05-09 22:05:14,952 INFO    Thread-4  :1554373 [wandb_init.py:init():715] updated telemetry
2024-05-09 22:05:14,954 INFO    Thread-4  :1554373 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-05-09 22:05:15,149 INFO    Thread-4  :1554373 [wandb_run.py:_on_init():2357] communicating current version
2024-05-09 22:05:15,179 INFO    Thread-4  :1554373 [wandb_run.py:_on_init():2366] got version response upgrade_message: "wandb version 0.17.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-05-09 22:05:15,180 INFO    Thread-4  :1554373 [wandb_init.py:init():799] starting run threads in backend
2024-05-09 22:05:15,519 INFO    Thread-4  :1554373 [wandb_run.py:_console_start():2335] atexit reg
2024-05-09 22:05:15,520 INFO    Thread-4  :1554373 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-05-09 22:05:15,520 INFO    Thread-4  :1554373 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-05-09 22:05:15,520 INFO    Thread-4  :1554373 [wandb_run.py:_redirect():2280] Redirects installed.
2024-05-09 22:05:15,521 INFO    Thread-4  :1554373 [wandb_init.py:init():842] run started, returning control to user process
2024-05-09 22:17:22,700 INFO    Thread-4  :1554373 [wandb_run.py:_finish():2064] finishing run vb2386/Sweep_Nerf/cge0nt76
2024-05-09 22:17:22,701 INFO    Thread-4  :1554373 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 1
2024-05-09 22:17:22,701 INFO    Thread-4  :1554373 [wandb_run.py:_restore():2287] restore
2024-05-09 22:17:22,701 INFO    Thread-4  :1554373 [wandb_run.py:_restore():2293] restore done
